lapis2/src/main/kotlin/org/genspectrum/lapis/model/VariantQueryCustomListener.kt

```diff
@@ -0,0 +1,123 @@
+package org.genspectrum.lapis.model
+
+import VariantQueryBaseListener
+import VariantQueryParser.Aa_insertionContext
+import VariantQueryParser.Aa_mutationContext
+import VariantQueryParser.AndContext
+import VariantQueryParser.Gisaid_clade_lineage_queryContext
+import VariantQueryParser.MaybeContext
+import VariantQueryParser.N_of_queryContext
+import VariantQueryParser.Nextclade_pangolineage_queryContext
+import VariantQueryParser.Nextstrain_clade_queryContext
+import VariantQueryParser.NotContext
+import VariantQueryParser.Nucleotide_insertionContext
+import VariantQueryParser.Nucleotide_mutationContext
+import VariantQueryParser.OrContext
+import VariantQueryParser.Pangolineage_queryContext
+import org.antlr.v4.runtime.tree.ParseTreeListener
+import org.genspectrum.lapis.silo.And
+import org.genspectrum.lapis.silo.Maybe
+import org.genspectrum.lapis.silo.NOf
+import org.genspectrum.lapis.silo.Not
+import org.genspectrum.lapis.silo.NucleotideSymbolEquals
+import org.genspectrum.lapis.silo.Or
+import org.genspectrum.lapis.silo.PangoLineageEquals
+import org.genspectrum.lapis.silo.SiloFilterExpression
+
+class VariantQueryCustomListener : VariantQueryBaseListener(), ParseTreeListener {
+    private var expressionStack = ArrayDeque<SiloFilterExpression>()
+
+    fun getExpr(): SiloFilterExpression {
+        return expressionStack.first()
+    }
+
+    override fun enterNucleotide_mutation(ctx: Nucleotide_mutationContext?) {
+        if (ctx == null) {
+            return
+        }
+        val position = ctx.position().text.toInt()
+        val secondSymbol = if (ctx.ambigous_nucleotide_symbol() != null) ctx.ambigous_nucleotide_symbol().text else "-"
```

Comment:
Can't we use the Elvis operator here?
```suggestion
        val secondSymbol = ctx.ambigous_nucleotide_symbol().text ?: "-"
```